src/Test/Perf/infra/fetch_build.csx

```diff
@@ -29,9 +35,16 @@ if (latestBuild == null)
 var latestBuildFolder = Path.Combine(sourceFolder, latestBuild);
 Log($"Fetching build \"{latestBuildFolder}\".");
 
-Directory.Delete(destinationFolder, recursive: true);
+if (Directory.Exists(destinationFolder))
+{
+    Directory.Delete(destinationFolder, recursive: true);
+}
 
-CopyDirectory(latestBuildFolder, destinationFolder);
+var result = ShellOut("Robocopy", $"/np /s {latestBuildFolder} {destinationFolder}", logStreamingOutput: true);
```

Comment:
RoboCopy has it's own set of return codes, and ```0``` is actually one that I would consider a failure in this scenario (no files copied).  We could plumb through an expected return code to ```ShellOutVital```, but I'm not inclined to do that right now, because I think RoboCopy is an oddball case.